_posts/2018-09-19-faq.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "LIVE4 FAQ"
+author: "LIVE4"
+---
+
+![My helpful screenshot](/assets/faq_header.jpg)
+
+Hey guys!
+
+Here we tried to answer most common questions regarding the LIVE4 app.
+
+# Permissions
+
+Since Facebook does not let external apps know the privacy level that user selects for post permissions, we can not make your stream visible to all of your friends if you have selected "Only me" visibility initially. 
+
+However, changing visibility permissions works perfectly on downgrading: from "Public" to "Friends", "Friends" to "Only me". 
+
+So we highly recommend to choose "Public" publish action permission from the very beginning. Please remember that you can always change it before each stream.
+
+To change the default publish permission: open Facebook via web, go to _Settings_ -> _Apps and websites_ -> _edit LIVE4 GoPro_ -> _"App visibility and post audience"_ permission
```

Comment:
To change the default publish permission: open Facebook via web, go to _Settings_ -> _Business integrations_ -> _edit LIVE4 GoPro_ -> _"App visibility and post audience"_ permission